### PR TITLE
Collect application logs

### LINKED ISSUE #41: Collect chat logs
I need to collect chat logs during the stream for further analysis

### PR DESCRIPTION
**Pull request by rexim**
Close #41

### CODE REVIEW

File: src/tsoder_app.erl [hunk 1]
```diff
@@ -14,7 +14,19 @@
 %% API
 %%====================================================================
 
+log_file_name(Suffix) ->
+    string:join(["log_", Suffix, ".txt"], "").
```
Comment (ForNeVeR) [inline src/tsoder_app.erl]: According to [the manual](http://erlang.org/doc/man/string.html), this function is obsolete and should be replaced with `lists:join`.

Comment (rexim) [inline src/tsoder_app.erl]: @ForNeVeR thanks for you comment! Unfortunately, `lists:join` in the current version of Erlang we support (18). I'll leave a comment to replace it with `lists:join` when Erlang version is > 18.

Comment (ForNeVeR) [inline src/tsoder_app.erl]: Oh, okay.

Comment (rexim) [inline src/tsoder_app.erl]: Captured the entire situation in #57 and #58

**Pull request merged**